.eslintrc

```diff
@@ -24,8 +24,10 @@
         "quotes": [2, "single"],
         "wrap-iife": 2,
         "no-console": 0,
+        "indent": ["error", 4],
```

Comment:
We should ask in the team which option we prefer, tabs, 4 spaces,2 spaces or other option.